wolfssl/ssl.h

```diff
@@ -34,6 +34,7 @@
 #include <wolfssl/version.h>
 #include <wolfssl/wolfcrypt/logging.h>
 #include <wolfssl/wolfcrypt/asn_public.h>
+#include <wolfssl/wolfcrypt/hash.h>
```

Comment:
I was a little hesitant to add a header include, but there are already dependencies in wolfcrypt so I went ahead with this solution.